src/server/helpers/critiquebrainz.ts

```diff
@@ -0,0 +1,18 @@
+import request from "superagent";
+
+export const getReviewsFromCB = async (bbid: string, entityType: string) => {
+	const mapEntityType = {
+		EditionGroup: "bb_edition_group",
+	};
+	entityType = mapEntityType[entityType];
+    console.log(entityType);
+	const res = await request
+        .get("https://beta.critiquebrainz.org/ws/1/review")
+        .query({
+            entity_id: bbid,
+            entity_type: entityType,
+            limit: 10,
+            offset: 0,
+        });
+    return res.body;
```

Comment:
This will need some error handling.
You'll need to do a few things: 
1. wrap this block (lines 9-17) in a try-catch block
2. I think (the docs aren't very clear for async/await syntax) that you might need to only return the body if `res.ok` is truthy 
3. if `res.ok` is falsy you can throw an error; not sure how you get the error message though
4. In the catch block you can log any error and return…judging from [this line](https://github.com/metabrainz/bookbrainz-site/pull/862/files#diff-1e08379bd6206ad9c3911bf9d48d1977e284ffc53ae218928448a2f197bdd8e9R47) an empty array?


It is possible that superagent throws errors for successful requests with an error http status, in which case steps 2 and 3 aren't needed.